### PR TITLE
Fpgacount2config

### PR DESCRIPTION
**Pull request by afswarm**

### CODE REVIEW

File: fpga.chart.py [hunk 1]
```diff
@@ -0,0 +1,147 @@
+# -*- coding: utf-8 -*-
+# Description: example netdata python.d module
```
Comment (sdressler) [inline fpga.chart.py]: Can you fill in lines 2, 3 and 4?

File: fpga.chart.py [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+             ]
+}
+MAX_DEFINITION = {
+    'options': [None, 'Max outstanding jobs', 'max oustanding', 'fpga', 'fpga', 'line'],
+    'lines': [['max_outstanding_compression_jobs', 'compression', 'absolute'],
+              ['max_outstanding_decompression_and_filter_jobs', 'decompress and filter', 'absolute'],
+              ['max_outstanding_filter_jobs', 'filter', 'absolute']
+             ]
+}
+TEMP_DEFINITION = {
```
Comment (sdressler) [inline fpga.chart.py:39]: Please remove

**Review by sdressler — CHANGES_REQUESTED** [review]
Few minor things...

File: fpga.chart.py [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
-# Description: example netdata python.d module
-# Author: Put your name here (your github login)
-# SPDX-License-Identifier: GPL-3.0-or-later
+# Description: python module to read data from loca FPGAs
```
Comment (sdressler) [inline fpga.chart.py]: Typo: "local"

File: fpga.chart.py [hunk 4]
```diff
@@ -1,7 +1,7 @@
...
+# Author: Luc Vlaming, Ashley Fraser
```
Comment (sdressler) [inline fpga.chart.py]: Please add me as well, the original version is mine.

File: fpga.chart.py (hunk 2 above)
Comment (afswarm) [reply fpga.chart.py:39]: remove TEMP_DEFINITION?

File: fpga.chart.py (hunk 1 above)
Comment (afswarm) [reply fpga.chart.py]: done

**Review by sdressler — COMMENTED** [review]
Sorry, this question just occurred to me: what happens when TEMP cannot be collected, i.e. will it die or just report "0"?

File: fpga.chart.py (hunk 3 above)
Comment (afswarm) [reply fpga.chart.py]: That is fixed in the other commit

File: fpga.chart.py (hunk 4 above)
Comment (afswarm) [reply fpga.chart.py]: Done

**Comment by afswarm** [comment]
> Sorry, this question just occurred to me: what happens when TEMP cannot be collected, i.e. will it die or just report "0"?



> Sorry, this question just occurred to me: what happens when TEMP cannot be collected, i.e. will it die or just report "0"?

It should report 0

**Review by sdressler — APPROVED** [review]

**Pull request merged**